cypress/fixtures/listOfArticles.json

```diff
@@ -1,24 +1,60 @@
+      },
+      {
+          "id": 3,
+          "title": "New Year 2020: 10 Moments This Decade That Made Us Say, \"Wait, What?\"",
+          "teaser": "Happy New Year 2020: This list of ten moments includes not just personalities in India and across the world, but also certain events which have had a number of consequences for those involved.",
+          "date": "2021-05-19, 15:10",
+          "category": "Science",
+          "image": "https://fake-news-2021-production.s3.eu-north-1.amazonaws.com/ireivnxpyf00g1vleopddtyk6x7o?response-content-disposition=inline%3B%20filename%3D%22article_image.jpg%22%3B%20filename%2A%3DUTF-8%27%27article_image.jpg&response-content-type=image%2Fjpeg&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAUYRJY36EVBTLAPKU%2F20210520%2Feu-north-1%2Fs3%2Faws4_request&X-Amz-Date=20210520T184424Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=81c353a6dcebea138f669ff6c5a93ffb68b6049dc663ec0686ecbc960ad54ad3"
+      },
+      {
+          "id": 2,
+          "title": "‘It is a trap!’: Inside the QAnon attack that never happened",
+          "teaser": "Why fears of violence on March 4, the mythical day Trump was supposed to be inaugurated to a second term, proved unfounded.",
+          "date": "2021-05-19, 15:10",
+          "category": "Illuminati",
+          "image": "https://fake-news-2021-production.s3.eu-north-1.amazonaws.com/sop4xummulpqmhcd1fxtk6yi5550?response-content-disposition=inline%3B%20filename%3D%22article_image.jpg%22%3B%20filename%2A%3DUTF-8%27%27article_image.jpg&response-content-type=image%2Fjpeg&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAUYRJY36EVBTLAPKU%2F20210520%2Feu-north-1%2Fs3%2Faws4_request&X-Amz-Date=20210520T184425Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=b08f50b061072cf59b452699ede978b817776fad1fd4a2c0a474f194d9ad82d0"
+      },
+      {
+          "id": 1,
+          "title": "New Vaccine Conspiracy Theories Are Going Viral in Arabic",
+          "teaser": "Facebook has been criticized for failing to curb misinformation in English. But little attention has been paid to the scale of the problem in Arabic.",
+          "date": "2021-05-19, 15:10",
+          "category": "Aliens",
+          "image": "https://fake-news-2021-production.s3.eu-north-1.amazonaws.com/39se5vnkfpub6nhawzaui6n5olws?response-content-disposition=inline%3B%20filename%3D%22article_image.jpg%22%3B%20filename%2A%3DUTF-8%27%27article_image.jpg&response-content-type=image%2Fjpeg&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAUYRJY36EVBTLAPKU%2F20210520%2Feu-north-1%2Fs3%2Faws4_request&X-Amz-Date=20210520T184425Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=f5c9bdc7b4c6e266f15a9d3ee3288e33d0d3ba4525cf4cd5b641e5f4c80b0dbb"
+      }
```

Comment:
Same here. 4 spaces vs 2 spaces in the rest of the project.

Reply:
fixed